.github/workflows/development.yml

```diff
@@ -39,6 +39,14 @@ jobs:
         ruby-version: ${{matrix.ruby}}
         bundler-cache: true
     
+    - name: Install liburing
+      run: |
```

Comment:
```suggestion
    - name: Install liburing
      if: ${{ matrix.ruby == '3.1' && matrix.os == 'ubuntu' }}
      run: |
```

Reply:
So you are saying this didn't work?

Reply:
It turns out that works!
I was tired and used tabs for indentation and was getting errors. It's good now.